src/mono/wasm/runtime/marshal.ts

```diff
@@ -335,7 +341,7 @@ export class ManagedError extends Error implements IDisposable {
     }
 
     toString(): string {
-        return `ManagedError(gc_handle: ${(<any>this)[js_owned_gc_handle_symbol]})`;
+        return this.managedStack || this.message;
```

Comment:
js `.stack` doesn't contain the exception message, is it intended that this is only the stack and not message + stack

Reply:
It does on chrome

![image](https://user-images.githubusercontent.com/271576/190675391-a86e6d93-1b36-4865-96ae-94ef5147074d.png)

Reply:
But I will stick to stack without message